src/Nox.Types/Types/TimeZoneCode/TimeZoneCode.cs

```diff
@@ -6,5 +9,215 @@ namespace Nox.Types;
+                                                        "SBT",
+                                                        "SCT",
+                                                        "SDT",
+                                                        "SGT",
+                                                        "SLST",
+                                                        "SRET",
+                                                        "SRT",
+                                                        "SST",
+                                                        "SYOT",
+                                                        "TAHT",
+                                                        "THA",
+                                                        "TFT",
+                                                        "TJT",
+                                                        "TKT",
+                                                        "TLT",
+                                                        "TMT",
+                                                        "TRT",
+                                                        "TOT",
+                                                        "TVT",
+                                                        "ULAST",
+                                                        "ULAT",
+                                                        "UTC",
+                                                        "UYST",
+                                                        "UYT",
+                                                        "UZT",
+                                                        "VET",
+                                                        "VLAT",
+                                                        "VOLT",
+                                                        "VOST",
+                                                        "VUT",
+                                                        "WAKT",
+                                                        "WAST",
+                                                        "WAT",
+                                                        "WEST",
+                                                        "WET",
+                                                        "WIB",
+                                                        "WIT",
+                                                        "WITA",
+                                                        "WGST",
+                                                        "WGT",
+                                                        "WST",
+                                                        "YAKT",
+                                                        "YEKT"};
+
+
+    /// <summary>
+    /// <inheritdoc/>
+    /// </summary>
+    /// <returns></returns>
+    internal override ValidationResult Validate()
     {
+        var result = base.Validate();
+
```

Comment:
To handle creation from mixed or lower case, add following here:

        Value = Value.ToUpperInvariant();

Reply:
May you can do this in the From method overriding the base class

Reply:
I have overridden the setter of the Value object. From is static.